poucave/app.py

```diff
@@ -144,6 +152,43 @@ def override_params(self, params: Dict[str, Any]):
     return web.json_response(infos)
 
 
+@routes.get("/checks/{project}")
+async def project_checkpoints(request):
+    checks = request.app["poucave.checks"]
+    cache = request.app["poucave.cache"]
+
+    try:
+        selected = checks.get(**request.match_info)
+    except ValueError:
+        raise web.HTTPNotFound()
+
+    body = []
+    for check in selected:
+        result = cache.get(check.cache_key)
```

Comment:
Should we be handling overridden parameters from URLs?

Reply:
I don't think so... that would mean the parameters with the same names would have the same value on every check?